packages/playground/_stories/main/Panel/Panel.stories.ts

```diff
@@ -88,6 +91,64 @@ FixedPanel.args = {
 };
 FixedPanel.storyName = "Fixed Panel (Can't be Collapsed/Expanded)";
 
+export const StickyHeader = Template.bind({});
+StickyHeader.decorators = [
+	(story) => {
+	return html`
+<style>
+	#panel-${index+1}::part(content) {
+		max-height: 50px;
+	}
+	#second-panel-stickyHeader::part(content) {
+		max-height: 100px;
+	}
+</style>
+<div style="height: 250px; overflow: scroll;">
+	${story()}
+<br />
```

Comment:
indent

Reply:
agreed to not indent since it's breaking